numpy/random/mtrand/mtrand.pyx

```diff
@@ -2289,7 +2289,7 @@ cdef class RandomState:
 
         Parameters
         ----------
-        df : int or array_like of ints
+        df : float or array_like of floats
             Degrees of freedom, should be > 0 as of NumPy 1.10.0,
```

Comment:
Should current docs describe old versions of NumPy?

Reply:
Yes. Can use the `.. versionchanged::` directive to do this more formally. There should be some examples in this file.

Reply:
But only if it changed in the code, changes in documentation don't need a note.

Reply:
Projects that support old versions of NumPy, SciPy for instance, need to be aware of when such changes occurred.